### PR TITLE
Various cleanups

### PR DESCRIPTION
**Pull request by dajohi**
- Use markdown comment-style
- remove stray </i>
- autocomplete is not valid for checkbox

### CODE REVIEW

File: theme_override/base.html [hunk 1]
```diff
@@ -95,8 +95,8 @@
         {% endif %}
       </defs>
     </svg>
-    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="drawer" autocomplete="off">
```
Comment (jholdstock) [inline theme_override/base.html]: This file is a copy of the one provided by our mkdocs theme, hence why it is in the "theme_override" directory. We only made tiny changes to this file which were mostly for #470 and #598. These changes have now been included in the theme by default, so we will actually be removing the file soon, its already deleted in #601 

I therefore suggest we simply revert this change for now.

File: docs/index.md [hunk 2]
```diff
@@ -2,24 +2,24 @@
 
 ---
 
-## <img class="dcr-icon" src="/img/dcr-icons/decred-logo.svg" /></i> What is Decred?
+## <img class="dcr-icon" alt="logo" src="/img/dcr-icons/decred-logo.svg" /> What is Decred?
 Decred is a cryptocurrency, similar to Bitcoin, with a strong focus on community input, open governance and sustainable funding and development. It utilizes a hybrid "proof-of-work" and "proof-of-stake" mining system to ensure that a small group cannot dominate the flow of transactions or make changes to Decred without the input of the community. A unit of currency is called a 'decred' (DCR). To ensure the integrity of the currency and prevent people from making fraudulent transactions or creating their own coins, Decred uses a method of recording transactions known as a blockchain.
 
 ---
 
-<more information to link to Decred governance, tech and dev pages>
```
Comment (jholdstock) [inline docs/index.md]: To be honest I think we should just remove this comment and the comment below - it doesn't seem like whoever wrote them is going to come back and implement them.

**Comment by dajohi** [comment]
updated
